consensus/broadcast/shardChainMessenger_test.go

```diff
@@ -451,6 +508,43 @@ func TestShardChainMessenger_BroadcastBlockDataLeaderNilMiniblocksShouldReturnNi
 	assert.Nil(t, err)
 }
 
+// This function should return ErrMockMarshalizer from core.CalculateHash
```

Comment:
no need for comments here, test name should be explicit enough

Reply:
updated